README.md

```diff
@@ -9,7 +9,12 @@ $ npm --version
 # Step 2: Ensure that Node is installed on your system. You can check this by using:
 $ node --version
 
-# Step 3: Clone the repository to your local machine, and install it.
+# Step 3: Make sure you have ts-node installed on your system. If not, you can install it globally like so:
+$ npm install -g ts-node
```

Comment:
Is there a specific reason why you specified -g? Isn't the purpose of this setup for the program to run?

Reply:
For some reason, when I tried to install `ts-node` as a `dev-dependency`, I got the command unrecognized error whenever trying to run:
![image](https://github.com/avelynhc/til_tracker/assets/78865303/71c9bf9b-1e0c-43d8-afb0-428ef985fcd5)

Maybe it was not correctly adding it to the path, which is why I installed it globally.

I tried installing it again as a dev-dependecy after your  comment, which gave me the same error.
Therefore, I installed `cross-env` and wrote a custom script to explicitly add it to the path before executing it like so:

```
npm install cross-env --save-dev
```
**package.json**
![image](https://github.com/avelynhc/til_tracker/assets/78865303/f7e96840-0ab2-456f-b607-7b40cfb4e991)

After this, the user can do:
![image](https://github.com/avelynhc/til_tracker/assets/78865303/b0977bc2-9510-4fbc-8b7e-d0fe1d8c0202)

Reply:
@Amnish04 I understand your point. What I did was simply to add `"start": "ts-node src/index.ts"` under the `scripts` in `pacakge.json`.

When I run `ts-node` in my terminal, I get this. 
<img width="368" alt="Screenshot 2023-09-22 at 2 52 56 PM" src="https://github.com/avelynhc/til_tracker/assets/75185537/fe3e6e2b-06be-4a4a-ae61-189dbdaf68d0">

After added the above code, I can do this now. 
<img width="377" alt="Screenshot 2023-09-22 at 2 53 57 PM" src="https://github.com/avelynhc/til_tracker/assets/75185537/cfb76294-c272-46c9-868f-742f533503d2">

Can you update `package.json` and `README.md` accordingly to make sure every developer uses the same version of `ts-node`?

Thanks again.

Reply:
@avelynhc Yes, I know it is working when we run the command as part of a script. But the problem with this approach is that your tool won't be able to recongnize any options as `npm` would consider those are being passed to it.

For example,
![image](https://github.com/avelynhc/til_tracker/assets/78865303/20d3857a-5666-4c80-a1f3-7fcad5bb7e0f)

Therefore, user needs to be able to execute ts-node from commandline directly.

Please let me know if I misunderstood something.

Reply:
@Amnish04 
I have a couple of questions if you don't mind me asking:

1. In which scenarios does the user have to be able to execute `ts-node` directly from the command line when there is `npm start` option?
2. When you install `ts-node` globally, how can you ensure that every developer or contributor uses the same version of `ts-node`?

Regarding the issue you mentioned in the previous comment, the user is able to execute `ts-node` directly in the command line by using `node node_modules/.bin/ts-node src/index.ts examples.txt` like this.

<img width="572" alt="Screenshot 2023-09-22 at 4 28 09 PM" src="https://github.com/avelynhc/til_tracker/assets/75185537/84c90e79-5c68-4fd5-be9d-d57a0ddafbed">

Thank you.

Reply:
@avelynhc 
1. Whenever the user wants to use an option/flag like `-v`, `-h`, or `-s`, they need to use ts-node directly. If the use a script like:
```
npm start -v
```
The `-v` flag will be consumed by `npm` which will show `npm` version instead of passing it to our program.
**Output:**
![image](https://github.com/avelynhc/til_tracker/assets/78865303/191bdb8a-3258-4a2a-8eaf-11094cf553e3)

2. We can specify the version in the readme file installation instrcutions:
```
npm install -g ts-node@10.9.1
```